src/features/game/expansion/components/resources/beehive/Beehive.tsx

```diff
@@ -337,6 +352,29 @@ export const Beehive: React.FC<Props> = ({ id }) => {
                 </div>
               </div>
             </div>
+            {currentRate > 0 && !!secondsLeftUntilFull && (
+              <div className="text-xs mx-1 mb-2">
+                {`${t("beehive.rateInfo")}`.replace(
```

Comment:
@Polysemouse Is this rate per day or per full beehive? cuz Queen Bee harvest 2x faster

Reply:
Yes this is the number of honey produced per day.  If the beehive has boosts the rate will be highe.